db/migrate/20200919174716_devise_create_users.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
```

Comment:
Is there anything else we might want on this table, like a display/ username? We'll also probably want a flag for whether or not they're an admin.